### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by jacobweinstock**
- save rke state file to deliverables
- comment out all optional values in `rancher.tfvars.example` for consistency
- add direct docker commands to README
- remove some unneeded code in hack files

### CODE REVIEW

File: rancher.tfvars.example [hunk 1]
```diff
@@ -9,13 +9,13 @@ vm_name = "rancher"
...
 # (Optional) number of CPUs all nodes will be assigned. Defaults to 8192
-vm_ram = 4096
+# vm_ram = 4096
 
 # (Optional) Linux virtual machine domain name
-vm_domain = "example.org"
+# vm_domain = "example.org"
 
 # (Optional) SSH public key to add to all nodes authorized_keys file.
 ssh_public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/0u1rdJJLt4ycshZg7zsbMbK/nydyn4rYuK7UXTtoH53geMbsxxwGOTZMKaTA4bVgSKBEwybVodfEzEsnsBgigUN/0OXAos6mWosDj8rPNGZ8lr7Sefn5x6ZhESVIuwpDp+bEukCYp6V+FtNiL2U+iWM5eOpl1vvPNMcYXhBdU948RZrubMTa8QlyzSY1Go46gfmVtW+pdXLxGM2sTXlKTA2IxMabmF/BxbVmoQPCNoFObtGxTBNfytbv4yZN73xq52xOvwhiOiLEo3Kj3vbjP49L750oaKKRIP62cd0/dU4y3r8OyJSJehpSYQhwy5slUjIl3KI/QRE8yuTX3+kV"
```
Comment (bgentry-ntap) [inline rancher.tfvars.example:21]: comment needed here. Also realized some parameters are not marked as (Optional) and probably should be with commented out examples. 

resource pool, server url, auto dns, all the bootstrap examples

File: README.md [hunk 2]
```diff
@@ -96,6 +96,16 @@ make rancher-destroy
 also clean up/remove the deliverables files generated from that run.  As such, if for some reason you'd like to save
 files from a previous run you'll want to copy them to another location.
 
+```bash
+# direct docker command to create the cluster
+docker run -it --rm $[PWD}/rancher.tfvars:/terraform/vsphere-rancher/terraform.tfvars -v ${PWD}/deliverables:/terraform/vsphere-rancher/deliverables netapp/ez-rancher:v0.1.0 apply -auto-approve
```
Comment (bgentry-ntap) [inline README.md:101]: should we just use the latest tag/remove the tag here?

File: rancher.tfvars.example (hunk 1 above)
Comment (jacobweinstock) [reply rancher.tfvars.example]: ah yes, thanks. updated.

File: README.md (hunk 2 above)
Comment (jacobweinstock) [reply README.md]: yeah, updated.

File: rancher.tfvars.example [hunk 3]
```diff
@@ -9,16 +9,16 @@ vm_name = "rancher"
 vm_template_name = "bionic-server-cloudimg-amd64"
 
 # (Optional) number of CPUs all nodes will be assigned. Defaults to 4
-vm_cpu = 2
+# vm_cpu = 2
```
Comment (hulquest) [inline rancher.tfvars.example:12]: Do we know who proposed the default value of 4 cpu and 8 GB ram values?  I think we've been using the 2 cpu/4 ram combo which seems to be working fine.

File: rancher.tfvars.example [hunk 4]
```diff
@@ -9,16 +9,16 @@ vm_name = "rancher"
...
 
 # (Optional) number of CPUs all nodes will be assigned. Defaults to 8192
```
Comment (hulquest) [inline rancher.tfvars.example:14]: Should clean up the description '(Optional) amount of RAM...'

Comment (jacobweinstock) [reply rancher.tfvars.example:14]: sorry, not following. clean up how?

File: rancher.tfvars.example (hunk 3 above)
Comment (jacobweinstock) [reply rancher.tfvars.example:12]: i dont recall how 4 cpu 8gb of memory became the default

File: rancher.tfvars.example (hunk 4 above)
Comment (hulquest) [reply rancher.tfvars.example:14]: The comment for the RAM configuration is a copy/paste from the CPU description on line 11.

File: rancher.tfvars.example (hunk 3 above)
Comment (bgentry-ntap) [reply rancher.tfvars.example:12]: Should be 2 vCPU and 8 GB per the min rancher requirements: https://rancher.com/docs/rancher/v2.x/en/installation/requirements/#cpu-and-memory

File: rancher.tfvars.example (hunk 4 above)
Comment (jacobweinstock) [reply rancher.tfvars.example:14]: oh duh, i see it now. thanks. will update.

File: rancher.tfvars.example (hunk 3 above)
Comment (hulquest) [reply rancher.tfvars.example:12]: > Should be 2 vCPU and 8 GB per the min rancher requirements: https://rancher.com/docs/rancher/v2.x/en/installation/requirements/#cpu-and-memory

This is the way.  Thank you @bgentry-ntap

**Pull request merged**